### PR TITLE
Get url support

### PR DESCRIPTION
**Pull request by sandeep-paliwal**
Added script to return Action URLs
1) For all actions in manifest
2) Specific action using 'action' option
3) CDN URLs for actions using 'cdn' flag

## Description
Return action URLs for
1) For all actions in manifest
2) Specific action using 'action' option
3) CDN URLs for actions using 'cdn' flag

## Motivation and Context

Support get-url command in aio app plugin

## How Has This Been Tested?

unit tests, and via CLI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x ] I have read the **CONTRIBUTING** document.
- [ x] I have added tests to cover my changes.
- [x ] All new and existing tests passed.

### CODE REVIEW

File: scripts/get.url.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    const urls = {}
+    if (options.action) {
+      const action = this.config.manifest.package.actions[options.action]
+      if (!action) {
+        throw new Error(`No action with name ${options.action} found`)
+      }
+      this.config.manifest.package.actions = {}
+      this.config.manifest.package.actions[options.action] = action
+    }
+    const actionUrls = await utils.getActionUrls(this.config, true)
```
Comment (moritzraho) [inline scripts/get.url.js:29]: shouldn't we also return the ui url? or as part of a later PR

Comment (sandeep-paliwal) [reply scripts/get.url.js]: For now, get.url.js is doing this ie. return both URLs. We can keep the utils function unchanged to return only URLs relevant as per the params passed to it.

File: scripts/get.url.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    urls.runtime = actionUrls
+    if (options.cdn) {
+      const cdnUrls = await utils.getActionUrls(this.config, false)
+      urls.cdn = cdnUrls
+    }
+    this.emit('end', taskName)
```
[the remark this answers is not in the document]
Comment (sandeep-paliwal) [reply scripts/get.url.js]: done

File: scripts/get.url.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  async run (options = {}) {
+    const taskName = 'GetUrl'
+    this.emit('start', taskName)
+    const urls = {}
+    if (options.action) {
+      const action = this.config.manifest.package.actions[options.action]
+      if (!action) {
+        throw new Error(`No action with name ${options.action} found`)
+      }
+      this.config.manifest.package.actions = {}
```
[the remark this answers is not in the document]
Comment (sandeep-paliwal) [reply scripts/get.url.js]: done

File: scripts/get.url.js (hunk 1 above)
Comment (moritzraho) [reply scripts/get.url.js]: No that's fine, returning the two urls should be ok, but I would additionally return the UI/webassets url

Comment (moritzraho) [reply scripts/get.url.js]: [ closing the loop ] UI urls to be added to a later PR

**Pull request merged**